MANIFEST

```diff
@@ -857,6 +861,24 @@ t/babel/numprints.xml
 t/babel/page545.pdf
 t/babel/page545.tex
 t/babel/page545.xml
+t/beamer/cover.pdf
+t/beamer/cover.tex
+t/beamer/cover.xml
+t/beamer/enumerate.pdf
+t/beamer/enumerate.tex
+t/beamer/enumerate.xml
+t/beamer/frames.pdf
+t/beamer/frames.tex
+t/beamer/frames.xml
+t/beamer/fsop.pdf
+t/beamer/fsop.tex
+t/beamer/fsop.xml
+t/beamer/overlayenv.pdf
+t/beamer/overlayenv.tex
+t/beamer/overlayenv.xml
+t/beamer/titles.pdf
+t/beamer/titles.tex
+t/beamer/titles.xml
```

Comment:
Too many tests! Good to be thorough, but for long term maintenance it is more convenient to combine into a single (or few) tests. So long as they are compatible as far as options or declarations are concerned, it is nice to work in a single test case through progressively more exotic features.

Reply:
Still too many tests. While more unit-like tests make good sense while developing, for maintenance (imho) they provide clutter and slowness. In most cases, if anything changes, they'll all fail anyway. Might as well have one (or very few) to focus on.  It looks to me like all the tests (w/ possible exception of fsop) are things that would be in a typical beamer doc, so might as well put them together; possibly separated by sections about what feature is being tested.